cucumber/authenticators/features/authn_status.feature

```diff
@@ -107,13 +107,21 @@ Feature: Authenticator status check
     And the authenticator status check fails with error "UserNotAuthorizedInConjur: CONJ00006E"
 
   Scenario: An authenticator without an implemented status check returns 503
-    Given I login as "alice"
+    Given a policy:
+    """
+    - !user alice
+    """
+    And I login as "alice"
```

Comment:
@micahlee can you explain why this is part of the PR? is this change related to the Rails 5 upgrade?

Reply:
Yes. When we started debugging test failures we discovered that we had quite a few scenarios that assumed a deterministic order, and that there was database state from prior runs.

We have started correcting this by doing more to ensure that each scenario "starts from scratch" and defines the entire scenario context itself.

These two scenarios are examples that assumed the user `alice` was already created in the database. So they now just need to specify that explicitly in the scenario by defining the user in policy.

Reply:
Nice, thanks!

When you create a PR to master please write this in a commit so it's clear what the change is . thanks for this!